### DISCUSSION TITLE
Ready Button Issues

### DISCUSSION THREAD

**Discussion by KanadeChii** [mo3rfan/syncplayer#21] [Q&A] [answered]
Hello, I have a Realme 6i running on android version 10. I downloaded the app yesterday and I can't seem to get it to work because the " Not Ready " button cannot be pressed. I have no way of making my user ready in order for the host to start the video.

**Comment by mo3rfan** [comment]
You need to load a video first so the button becomes clickable. Have you done that?

**Comment by KanadeChii** [comment, marked as answer]
The host is on a windows desktop.

Im sorry, I didnt realize that both devices must have the same video file.
I thought that syncplay would broadcast a video file to each connected
individual inside the room. I will try again later with both devices
containing the same video file

**Reply by mo3rfan** [reply]
Yes the app does not broadcast video. Only the video **states**. :)

**Comment by KanadeChii** [comment]
Ok, thanks for the clarification.
